cli/state.go

```diff
@@ -1527,6 +1529,22 @@ var StateSectorCmd = &cli.Command{
 		}
 		fmt.Println()
 
+		// Display DealIDs - try both deprecated field and new ProviderSectors method
+		dealIDs := si.DeprecatedDealIDs
+		fmt.Printf("DealIDs (deprecated): %v\n", dealIDs)
+
+		// For actors v13+, try to get deal IDs from market actor's ProviderSectors HAMT
+		if nv >= network.Version13 {
```

Comment:
Actors version and network version are not the same thing, so you'll need to do this differently.

The mapping is available here: https://github.com/filecoin-project/go-state-types/blob/571b84617a4b7fe032cf63c25e0c079f90e2f8a7/actors/version.go#L32

But, you probably just want to switch to checking for `network.Version22` rather than handling mapping. Something like this:

```suggestion
		// Deals were moved into market actor's ProviderSectors in NV22 / Actors v13
		if nv >= network.Version22 {
```

Reply:
Thanks, Feedback has been addressed in recent commit